k8s/microservices/account.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
```

Comment:
why are we applying these manifests when initializing the cluster? the pipelines in each service repo should be applying these manifests, so why does this repo's pipeline care?
i mean the ./microservices manifests - the ./prerequisites manifests are fine.

Reply:
The idea with this pipeline is to initialize a new cluster and prepare for individual microservice updates afterward.  It does not handle the shared cluster resources, which we intend to cover in our documentation.

Reply:
We are actually working with Sam too right now and we may be able to add the Istio setup to this initialization pipeline too.

Reply:
it's preferable to let each service's pipeline handle the initial deployment of itself as well as the updates later on, but if it works we can keep it